link-grammar/link-includes.h

```diff
@@ -113,6 +113,8 @@ link_public_api(void)
      dictionary_set_data_dir(const char * path);
 link_public_api(char *)
      dictionary_get_data_dir(void);
+link_public_api(char *)
+     linkgrammar_get_data_dir(void);
```

Comment:
Hmm. I merged this pull req, but I'm thinking that this routine should probably be merged with `dictionary_get_data_dir()` -- that is, once the location of the dictionary has been determined, that is what `dictionary_get_data_dir` should return.